lib/bulk_data_test_kit/v2.0.0/bulk_data_export_cancel_test.rb

```diff
@@ -16,6 +16,8 @@ class BulkDataExportCancelTest < Inferno::Test
         http://hl7.org/fhir/uv/bulkdata/STU2/export.html#bulk-data-delete-request
       DESCRIPTION
 
+      verifies_requirements 'hl7.fhir.uv.bulkdata_2.0.0@122'
```

Comment:
See new requirements at end of bulk data requirements. 308 might work here? Reread the original text to confirm.